### ISSUE TITLE
openvpn ignores up/down scripts, the replacement /usr/libexec/openvpn-hotplug ignores fundamental parameters

### ISSUE THREAD

**Issue by ezplanet** [openwrt/packages#15121]
Maintainer: Felix Fietkau  @ndb168 
Environment: Any - OpenWRT 21.02

Description: openvpn in OpenWRT 21.02 up/down scripts configuration is ignored

It appears that in OpenWRT 21.02 openvpn was changed to ignore up down scripts configuration.
It appears that undocumented script /usr/libexec/openvpn-hotplug replaced this configuration.
This script in turn picks only one parameter of the 7 passed by openvpn (the instance name) and ignores all the other parameters that are fundamental to setting up, for example, ip rules and ip routes on vpn connection.

Please see the following forum thread for a detailed explanation:
https://forum.openwrt.org/t/openwrt-21-02-openvpn-client-up-down-scripts-configuration-ignored/91126

**Comment by ezplanet** [comment]
Please see my proposed fix in the following PR:
https://github.com/openwrt/packages/pull/15124

**Comment by ezplanet** [comment]
Initially I followed the the hotplug scripts flow starting with /usr/libexec/openvpn-hotplug. Following a more in depth analysis I realized that it is sufficient to amend /etc/hotplug.d/openvpn/01-user that tries and fails to get up/down scripts configuration from the translated ovpn instance configuration. Those parameters are ignored in the translation from the uci openvpn configuration and thus are missing
This proposed change simply takes the parameters from the uci configuration instead.

https://github.com/openwrt/packages/pull/15125
